cortex/quickflat/utils.py

```diff
@@ -76,7 +78,16 @@ def make_flatmap_image(braindata, height=1024, recache=False, **kwargs):
         badmask = np.array(pixmap.sum(1) > 0).ravel()
         img = (np.nan*np.ones(mask.shape)).astype(data.dtype)
         mimg = (np.nan*np.ones(badmask.shape)).astype(data.dtype)
-        mimg[badmask] = (pixmap*data.ravel())[badmask].astype(mimg.dtype)
+
+        if nanmean:
+            # create nanmean of pixmap*data
+            nan_to_num_mean = pixmap * np.nan_to_num(data.ravel())
```

Comment:
I find this code rather hard to follow.
One key is that `pixmap*data` is actually a dot product, since `pixmap` is sparse.
I would write it `pixmap.dot(data)` for clarity.
Another key is that `pixmap` contains weights summing to one, which explain why you do not need to divide by `pixmap.sum(1)` in the default case. I would add a comment to clarify that.